dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -49,11 +48,14 @@ def rubric_params
       :level_id,
       :lesson_id,
       learning_goals_attributes: [
+        :id,
```

Comment:
I don't think you want to include the id here. In particular, we want MySQL to assign ids. If you need this for update, can you remove setting the ID on the client?

Reply:
Now that I'm thinking more about this -- I could see why you would need the id to be passed back to update the correct row. But now I see that you're already removing them!